src/api.rs

```diff
@@ -469,30 +471,34 @@ impl Events {
     /// ```
     #[inline]
     pub fn events(&self) -> EventIterator {
+        let ptr = self.events.as_ptr() as *const *const Event;
         EventIterator {
-            events: self.events_raw(),
-            index: 0,
+            current: ptr,
+            end: unsafe { ptr.offset(self.num_events as isize) },
+            _marker : PhantomData
```

Comment:
extremely nitpickey, but keep consistent spacing around the `:` (seems like the crate tends towards no space before it, one space after)

Reply:
[resolved](https://github.com/RustAudio/vst-rs/pull/93/commits/648f0288d4b2ffd61f09859e2432304a859ef09d)